.ci/common/logstash.Dockerfile

```diff
@@ -1,9 +1,11 @@
 ARG ELASTIC_STACK_VERSION
 FROM docker.elastic.co/logstash/logstash:$ELASTIC_STACK_VERSION
-COPY --chown=logstash:logstash Gemfile /usr/share/plugins/plugin/
-COPY --chown=logstash:logstash *.gemspec /usr/share/plugins/plugin/
+USER logstash
+COPY --chown=logstash:logstash Gemfile /usr/share/plugins/plugin/Gemfile
+COPY --chown=logstash:logstash *.gemspec VERSION* version* /usr/share/plugins/plugin/
 RUN cp /usr/share/logstash/logstash-core/versions-gem-copy.yml /usr/share/logstash/versions.yml
-ENV PATH="${PATH}:/usr/share/logstash/vendor/jruby/bin"
+# NOTE: since 8.0 JDK is bundled as part of the LS distribution under $LS_HOME/jdk
```

Comment:
It's from `7.10` that's bundled in the distribution

Reply:
it is copied from .ci/Dockerfile I can change it to 7.10